website/package.json

```diff
@@ -78,20 +78,20 @@
     "ember-concurrency": "^4.0.2",
     "ember-fetch": "^8.1.2",
     "ember-get-config": "^2.1.1",
-    "ember-load-initializers": "^2.1.2",
-    "ember-math-helpers": "^3.0.0",
+    "ember-load-initializers": "^3.0.1",
+    "ember-math-helpers": "^4.2.1",
     "ember-meta": "^2.0.0",
     "ember-modifier": "^4.1.0",
     "ember-page-title": "^8.2.3",
     "ember-power-select": "^8.2.0",
-    "ember-prism": "^0.13.0",
+    "ember-prism": "^1.0.0",
```

Comment:
noticed that this package is now archived, so we create a ticket in our backlog to investigate other options...

Reply:
here is better alternative I recently used to replace `ember-prism` https://github.com/IgnaceMaes/ember-shiki

Reply:
I could have been more clear with my comment: I think the upgrade is fine but down the road we'll need to look at other options potentially